docs/self-hosted-appcircle/configure-server/advanced-configuration/store-dist-dmz.md

```diff
@@ -0,0 +1,503 @@
+
+You will create all the configuration files on the Appcircle server and then move the created configuration files to the Appcircle DMZ server.
+
+To create the Appcircle DMZ server configuration, you can follow the steps below.
+
+- Login to the Appcircle server with SSH.
+
+- Go to the Appcircle server directory.
+
+```bash
+cd appcircle-server
+```
+
+- Stop the running Appcircle server.
+
+<SpacetechExampleInfo />
+
+```bash
+./ac-self-hosted.sh -n spacetech down
+```
+
+- Create the new configuration files for the Appcircle DMZ and the Appcircle server.
+
+```bash
+./ac-self-hosted.sh -n spacetech export --dmz
+```
+
+:::info
+The `--dmz` flag in the `export` subcommand above creates the configuration files according to your `global.yaml` file of your project for the both the Appcircle DMZ server and the Appcircle server.
+:::
+
+- Check the exported DMZ directory that contains the required files to create Appcircle DMZ server.
+
+```bash
+ls -lah projects/spacetech/export/dmz/
+```
+
+- Start the Appcircle server.
+
+```bash
+./ac-self-hosted.sh -n spacetech up
+```
+
+- Compress the directory into a tarball in the Appcircle server.
+
+```bash
+tar -czf dmz.tar.gz -C projects/spacetech/export/dmz/ .
+```
+
+- Transfer the `dmz.tar.gz` file to the Appcircle DMZ server with a file transfer protocol like `scp` or `ftp`.
+
+## Creating the Appcircle DMZ Server
+
+### Create Appcircle DMZ Directory
+
+You need to create a directory for the Appcircle DMZ server and extract the transferred configuration files.
+
+- Create a Appcircle DMZ server.
+
+```bash
+mkdir -p appcircle-server-dmz
+```
+
+- Extract the tarball you transferred from the Appcircle server.
+
+```bash
+tar -xzf dmz.tar.gz -C appcircle-server-dmz
+```
+
+- Change directory into the new directory.
+
+```bash
+cd appcircle-server-dmz
+```
+
+### Configure the System
+
+Install the required packages and configurations on the system.
+
+:::caution
+
+You need to have root access on your system for this step. Being able to run `sudo` is sufficient for the following step. (sudoer)
+
+Run the command without `sudo`. The script will ask for the user password if it's required.
+
+:::
+
+```bash
+./ac-self-hosted-dmz.sh -i
+```
+
+### Starting the Appcircle DMZ Server
+
+After you have configured the system with the steps above, you are ready to run the Appcircle DMZ server.
+
+- Start the Appcircle DMZ server.
+
+```bash
+./ac-self-hosted-dmz.sh up
+```
+
+- Check the health of the Appcircle DMZ services.
+
+```bash
+./ac-self-hosted-dmz.sh check
+```
+
+:::caution
+Be sure that all the services are running and healthy. If there are connection problem between the Appcircle DMZ server and the Appcircle server, the services won't be healthy.
+:::
+
+## Stopping the Appcircle DMZ Server
+
+If you need to stop the Appcircle DMZ server in a case, you can run the the command below:
+
+```bash
+./ac-self-hosted-dmz.sh down
+```
+
+## Upgrading Appcircle DMZ and Appcircle server
```

Comment:
We should have a note (caution or info box) at **Upgrade Server** page that's referencing here and  telling the importance of upgrading DMZ on every update keeping synced with the Appcircle server.
- https://deploy-preview-524--reverent-galileo-8ef035.netlify.app/self-hosted-appcircle/update

Reply:
👆 Above case should also be considered for the offline upgrade scenario.
- https://deploy-preview-524--reverent-galileo-8ef035.netlify.app/self-hosted-appcircle/configure-server/offline-installation